charts/headlamp/templates/clusterrolebinding.yaml

```
@@ -16,5 +16,5 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: {{ include "headlamp.serviceAccountName" . }}
-  namespace: kube-system
+  namespace: {{ .Release.Namespace | default "kube-system" }}
```

Good catch, I think the default value is not necessary just `.Release.Namespace` should do

```suggestion
  namespace: {{ .Release.Namespace }}
```